### PR TITLE
[doc](partial update) add flink-connector docs

### PR DESCRIPTION
**Pull request by zhannngchen**
## Proposed changes

Issue Number: close #xxx

add docs for how to enable partial update in flink connector

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhannngchen** [comment]
run buildall

File: docs/zh-CN/docs/data-operate/update-delete/partial-update.md [hunk 1]
```diff
@@ -116,6 +116,13 @@ partial_columns:true
 
 同时在`columns`中指定要导入的列（必须包含所有key列，不然无法更新）
 
+#### Flink Connector
+如果使用Flink Connector, 需要添加如下配置：
+```
+'sink.properties.partial_columns' = 'true',
+```
+同时在`sink.properties.column`中制定要导入的列（必须包含所有key列，不然无法更新）
```
Comment (liaoxin01) [inline docs/zh-CN/docs/data-operate/update-delete/partial-update.md:124]: 制定->指定

**Comment by zhannngchen** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Review by bobhan1 — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**